deploy/legacy/manifests/controller/helm/retina/values.yaml

```diff
@@ -41,6 +41,7 @@ enablePodLevel: false
 remoteContext: false
 enableAnnotations: false
 bypassLookupIPOfInterest: false
+dataAggregationLevel: 0
```

Comment:
i'm not a big fan of having this set as a number because it could be confusing later if we add different levels of data aggregation (0=low, 1=high, 2=med, etc) (not too sure if we are planning to do this). How about we have it as a string value instead of this numerical value? It maps to an enum anyways

Reply:
yeah that would make more sense, ill switch to that